docs/reference/commandline/manifest.md

```diff
@@ -0,0 +1,274 @@
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+```markdown
+Usage:  docker manifest COMMAND
+
+Manage Docker image manifests and manifest lists
+
+Options:
+      --help   Print usage
+
+Commands:
+  annotate    Add additional information to a local image manifest
+  create      Create a local manifest list for annotating and pushing to a registry
+  inspect     Display an image manifest, or manifest list
+  push        Push a manifest list to a repository
+
+```
+
+## Description
+
+The `docker manifest` command by itself performs no action. In order to operate
+on a manifest or manifest list, one of the subcommands must be used.
+
+A single manifest is information about an image, such as layers, size, and digest.
+The docker manifest command also gives users additional information such as the os
+and architecture an image was built for.
+
+A manifest list is a list of image layers that is created by specifying one or
+more (ideally more than one) image names. It can then be used in the same way as
+an image name in `docker pull` and `docker run` commands, for example.
+
+Ideally a manifest list is created from images that are identical in function for
+different os/arch combinations. For this reason, manifest lists are often referred to as
+"multi-arch images." However, a user could create a manifest list that points
+to two images -- one for windows on amd64, and one for darwin on amd64.
+
+### manifest inspect
+
+```
+manifest inspect --help
+
+Usage:  docker manifest inspect [OPTIONS] [MANIFEST_LIST] MANIFEST
+
+isplay an image manifest, or manifest list
+
+Options:
+      --help       Print usage
+      --insecure   allow communication with an insecure registry
+  -v, --verbose    Output additional info including layers and platform
+```
+
+### manifest create 
+
+```bash
+Usage:  docker manifest create MANFEST_LIST MANIFEST [MANIFEST...]
+
+Create a local manifest list for annotating and pushing to a registry
+
+Options:
+  -a, --amend   Amend an existing manifest list
+      --insecure   allow communication with an insecure registry
+      --help    Print usage
+```
+
+### manifest annotate
+```bash
+Usage:  docker manifest annotate [OPTIONS] MANIFEST_LIST MANIFEST
+
+Add additional information to a local image manifest
+
+Options:
+      --arch string               Set architecture
+      --help                      Print usage
+      --os string                 Set operating system
+      --os-features stringSlice   Set operating system feature
+      --variant string            Set architecture variant
+
+```
+
+### manifest push
+```bash
+Usage:  docker manifest push [OPTIONS] MANIFEST_LIST
+
+Push a manifest list to a repository
+
+Options:
+      --help    Print usage
+      --insecure   allow push to an insecure registry
+  -p, --purge   Remove the local manifest list after push
+```
+
+## Working with Insecure Registries
```

Comment:
Should this part go under the `## Description` heading? If so, increase its heading level one.

Use sentence  case (Working with insecure registries)

Reply:
yah, it probably makes sense that it's not its own section